### ISSUE TITLE
[kickasstorrent] Exception (kickasstorrent): Request headers must contain only ASCII characters.: Request headers must contain only ASCII characters. (Test)

### ISSUE THREAD

**Issue by lucknaumann** [Jackett/Jackett#9387]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed. ✔️

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug. ✔️

### Environment

**OS**: Windows 10 v1809 (*maybe someday I'll upgrade...*) 

**.Net Runtime**: [.Net-Core/.Net-Framework/Mono] Microsoft .NET Framework

**.Net Version**: version 4.0.30319.42000

**Jackett Version**: 0.16.1037.0

**Last Working Jackett Version**: I don't know as I decided to check my indexers today for some unknown reason.

**Are you using a proxy or VPN?** [yes/no] no

### Description

[List steps to reproduce the error and details on what happens and what you expected to happen]

I tested the indexer in Sonarr & when it failed I tested in the Jackett web GUI and when it failed it provided an error message:

````posh
An error occurred while testing this indexer
Exception (kickasstorrent): Request headers must contain only ASCII characters.: Request headers must contain only ASCII characters.
Click here to open an issue on GitHub for this indexer.
````

I followed the instructions to Click here... and here we are.

### Logged Error Messages

[Place any relevant error messages you noticed from the logs here.]

[Make sure you attach the full logs with your personal information removed in case we need more information]

> Make sure you remove your username/password/cookies

Not sure what the "cookies" would be in these files but removed whatever personal info I found.

### Screenshots

[Place any screenshots of the issue here if needed]

**Comment by garfield69** [comment]
duplicate of #7907
see https://github.com/Jackett/Jackett/issues/7907#issuecomment-605508904 for instructions.
Please report back success or failure.

**Comment by lucknaumann** [comment]
Hmm. That's weird. I searched the entire string and only came up with one **closed** issue that the person didn't fill out and was labeled "No details provided": #8852. I see now that I missed your response on that one. Then I searched just "kickasstorrents" and still didn't find those issues. Now I redid the search and see a ton. What the heck? And am I only one who actually filled out the info and provided the logs unnecessarily? 

Anyhow -- thanks for such a speedy response! Yeah, it worked the first time without having to use an alternate SiteLink. Thanks!

> Try the following:
> 
> use your Jackett dashboard to trashbin your current KickassTorrent config
> add a fresh config from the add-indexer page using the wrench icon
> see if that helps.
> If not, repeat the process but pick one of the alternate SiteLinks and see if that helps.
